packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -271,13 +272,15 @@ export default {
       // the placement prop when there is not enough available room for the tip
       // to display and it uses a fallback placement.
       currentPlacement: this.placement,
+
+      isTouchDevice: false,
     };
   },
 
   computed: {
     // whether the tooltip is visible or not.
     isVisible () {
-      return this.isShown && this.enabled && (!!this.message.trim() || !!this.$slots.default);
+      return this.isShown && this.enabled && (!!this.message.trim() || !!this.$slots.default) && !this.isTouchDevice;
```

Comment:
I would like to improve this computed property, could be something like this?

  ```
isVisible() {
    const hasMessage = !!this.message.trim();
    const hasDefaultSlot = !!this.$slots.default;
    const isDeviceCompatible = !this.isTouchDevice; 

    const shouldBeVisible = this.isShown && this.enabled && (hasMessage || hasDefaultSlot);

    return shouldBeVisible && isDeviceCompatible;
  },
```

I guess it will work as this way, double check it please to be sure, and make this update also in the vue3 version.

Reply:
Done